glass/core/array.py

```diff
@@ -166,7 +166,7 @@ def trapezoid_product(
     y = np.interp(x, *f)
     for f_ in ff:
         y *= np.interp(x, *f_)
-    return np.atleast_1d(np.trapezoid(y, x, axis=axis))
+    return np.trapezoid(y, x, axis=axis)  # type: ignore[no-any-return]
```

Comment:
Should the mypy fixes be another PR?

Reply:
I hadn't asked for a review... The PR was open to get the tests running. I've merged in #484.

Reply:
Ah, my bad!